### PR TITLE
add ids to the return response of start_plotting

### PR DESCRIPTION
**Pull request by dkackman**
This is a simple addition to collect the id's of plots as they are queued and return them with the response. Makes it a titch easier on the caller to manage the work.

### CODE REVIEW

**Comment by dkackman** [comment]
I fixed the one black message (trailing white space) related to my change. The default run of black reformatted some other stuff too, which I did not include. I can include those if need be.

File: chia/daemon/server.py [hunk 1]
```diff
@@ -538,8 +538,10 @@ async def start_plotting(self, request: Dict[str, Any]):
             }
             return response
 
+        ids : List[str] = []
```
Comment (aqk) [inline chia/daemon/server.py:541]:
```suggestion
        ids: List[str] = []
```

**Review by aqk — CHANGES_REQUESTED** [review]
Hi @dkackman, I think if you accept the suggested change, we can take your PR

**Comment by dkackman** [comment]
> Hi @dkackman, I think if you accept the suggested change, we can take your PR

Done

**Comment by dkackman** [comment]
hiya @aqk - I accepted the changes but it's blocked on what looks like a broken build action. Could you take a look and unstick this one?

**Comment by jack60612** [comment]
> hiya @aqk - I accepted the changes but it's blocked on what looks like an unrelated broken build action. Could you take a look and unstick this one?

i got it

**Comment by hoffmang9** [comment]
I'm afraid it may be to old to just rerun the test. Can you force push or make a trivial change to rerun them @dkackman ?

**Comment by dkackman** [comment]
> I'm afraid it may be to old to just rerun the test. Can you force push or make a trivial change to rerun them @dkackman ?

Done

**Comment by dkackman** [comment]
Hey guys. Anything more needed in this one?

**Comment by jack60612** [comment]
> Hey guys. Anything more needed in this one?

Sometimes greatness comes naturally

**Review by emlowe — APPROVED** [review]
seems ok to me

**Pull request merged**